src/package/packageVersionCreateRequestApi.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// Node
+import * as util from 'util';
+
+// Local
+import { Connection, Messages } from '@salesforce/core';
+import {
+  Package2VersionCreateRequestError,
+  Package2VersionCreateRequestResult,
+  PackagingSObjects,
+} from '../interfaces';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/packaging', 'messages');
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const packageUtils = require('../utils/packageUtils');
+const STATUS_ERROR = 'Error';
+const QUERY =
+  'SELECT Id, Status, Package2Id, Package2VersionId, Package2Version.SubscriberPackageVersionId, Tag, Branch, ' +
+  'CreatedDate, Package2Version.HasMetadataRemoved, CreatedById ' +
+  'FROM Package2VersionCreateRequest ' +
+  '%s' + // WHERE, if applicable
+  'ORDER BY CreatedDate';
+const ERROR_QUERY = "SELECT Message FROM Package2VersionCreateRequestError WHERE ParentRequest.Id = '%s'";
+const STATUSES = ['Queued', 'InProgress', 'Success', 'Error'];
+
+type PackageVersionCreateRequestApiOptions = {
+  createdlastdays?: number;
+  connection?: Connection;
+  status?: string;
+};
+
+export class PackageVersionCreateRequestApi {
```

Comment:
(not blocking) never liked this name.  I have ideas for how to rename but that can come later.